maas/data_source_maas_boot_source.go

```diff
@@ -12,7 +12,7 @@ import (
 func dataSourceMaasBootSource() *schema.Resource {
 	return &schema.Resource{
 		ReadContext: dataSourceMaasBootSourceRead,
-		Description: "Provides a resource to fetch the MAAS boot source.",
+		Description: "Provides a resource to manage the MAAS boot source.",
```

Comment:
this one needs to be reverted. The data source is what Terraform is giving us to fetch resources. The initial message was correct.

Reply:
In addition, the docs should be re-generated